css/templates/e-commerce/page-vars.css

```diff
@@ -29,10 +44,8 @@
 
   /* Narrow custom properties */
   --narrow-page-padding: var(--space-3);
-  --narrow-nav-item-height: 3.5rem;
-  --narrow-body-font-size: .9rem;
+  --navbar-height: 3.5rem;
 
   /* Wide custom properties */
-  --wide-nav-item-height: 5rem;
-  --wide-body-font-size: 1rem;
+  --wide-navbar-height: 5rem;
```

Comment:
It wasn't easy to define a different navheight for mobile and desktop, so I created a new page var and redefined in main css. Whenever I tried to wrap a CSS custom property declaration around a media query, postCSS would throw an error about vars not being defined in the `:root` scope.